scripts/loadLatestPremiumDockerImage.sh

```diff
@@ -0,0 +1,32 @@
+#!/usr/bin/env bash
```

Comment:
Let's align it with the rest of the scripts in `cloudify-ui-comon`:
* Use kebab case for filenames
* Add description in the same format inside the file
* Update `scripts/README.md` file

Reply:
Done.